clouddriver-lambda/src/main/java/com/netflix/spinnaker/config/LambdaServiceConfig.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2021 Armory, Inc.
```

Comment:
2023?

Reply:
Since this was a move... kept the original date.